backend/src/api/classes/dto/existing-class-teacher.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { ExistingClassDto } from "./existing-class.dto";
+import { ApiProperty } from "@nestjs/swagger";
+import { Exclude } from "class-transformer";
+
+export class ExistingClassTeacherDto extends ExistingClassDto {
```

Comment:
Can we maybe name this sth like "ClassWithTeacher", the name confused me at first ^^